clusters/hive/pools/hypershift/install-config-aws-us-east-1_secret.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: install-config-aws-us-east-1
+  namespace: hypershift-cluster-pool
+stringData:
+  # The public key is taken from https://github.com/openshift/release/blob/80d88fb8ab7eec427d001ed7fb67741251c49dcf/core-services/ci-secret-bootstrap/_config.yaml#L869-L871
```

Comment:
Are you using the same SSH key below?
You can remove it because you do not have the private key.

Reply:
Thanks, I just created a new key for us to use that we have ownership of.